src/main/java/core/basesyntax/operation/BalanceOperation.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.operation;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+
+public class BalanceOperation implements OperationHandler {
+    @Override
+    public void processOperation(FruitTransaction fruitTransaction) {
+        Storage.storage.put(fruitTransaction.getFruit(), fruitTransaction.getQuantity());
+    }
```

Comment:
Don't begin class or method implementation with empty line. There is an unnecessary empty line at the beginning of the class definition (line 6) and the method definition (line 7). Please remove these to adhere to the coding standards.